modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/representative_users_controller.rb

```diff
@@ -4,7 +4,48 @@ module AccreditedRepresentativePortal
   module V0
     class RepresentativeUsersController < ApplicationController
       def show
-        render json: @current_user
+        # TODO: Once we figure out how we're handling serialization and which library we're using,
+        # moving this serialization logic out to to a serialization layer.
+        render json: {
+          account: {
+            # NOTE: In regard to the in progress form system, this value is only
+            # showing up as error log metadata that identifies a user. To note
+            # is that the VA.gov-wide implementation is exposing the CSP user
+            # ID, while we are improving upon that by exposing the ARP user ID
+            # which will make investigating errors easier. However, it might
+            # confuse someone who is used to debugging the VA.gov-wide scenario.
+            account_uuid: @current_user.user_account_uuid
+          },
+          profile: {
+            first_name: @current_user.first_name,
+            last_name: @current_user.last_name,
+            verified:,
+            sign_in: {
+              service_name: @current_user.sign_in[:service_name]
+            }
+          },
+          prefills_available: [],
+          in_progress_forms:
+        }
+      end
+
+      private
+
+      def verified
+        # TODO: Should we make this a genuine derivation of the user's LOA? The
+        # source of truth of the fact that ARP's users are always verified is
+        # ARP's `SignIn::ClientConfig` which requires such a high LOA level.
+        true
```

Comment:
@bosawt It seems to me there is a calculation that is genuinely appropriate to make for determining LOA and verified and it just happens to be the case that all our users verified are as a consequence of our client config (is this correct?) and furthermore the single source of truth of that fact is expressed exactly in the client config (again if that's correct). Put another way, if the source of truth changed to enable unverified uses, this code would become logically out of sync.

Reply:
If my line is reasoning is correct, we can reintroduce all the logic at the right layers mirroring VA.gov-wide code that derives loa and verified from the source of truth (which I guess is actually the presence if `icn`).

Reply:
I think it's safe to check the `UserAccount.verified?` function if you'd prefer a dynamic result here. Yes, it's just checking for presence of `icn`, which matches the requirements for VA.gov (a user with credential_a will verify and this credential will never lose verification status. ARP can probably safely rely on these requirements too).